### ISSUE TITLE
Handle singular and plural in changelog

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#3113]
[On this product](https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=26044422), Photos were deleted. The changelog entries (at the bottom) are confusing
![image](https://user-images.githubusercontent.com/1689815/77634399-2874d180-6f51-11ea-8755-2b201d505213.png)
`March 26, 2020 at 11:00:43 AM CET - charlesnepote Uploaded images (Deleted: 5) - Moved images 5 to trash - view`
should be
`March 26, 2020 at 11:00:43 AM CET - charlesnepote Uploaded images (Deleted: 5) - Moved image 5 to trash - view`
because there's only one image being deleted

**Comment by VaiTon** [comment]
Yeah it's pretty confusing. Do we care about what index the deleted image had?
